packages/orderbook/test/js_api/gui.test.ts

```
@@ -902,7 +903,7 @@
 		it('should check if field is preset', async () => {
 			gui.saveFieldValue('test-binding', {
 				isPreset: true,
-				value: gui.getFieldDefinition('test-binding').presets[0].id
+				value: gui.getFieldDefinition('test-binding').presets![0].id
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)